build/darwin/create-universal-app.ts

```diff
@@ -32,15 +31,13 @@ async function main(buildDir?: string) {
 		'**/Credits.rtf',
 	];
 
-	const canAsar = isAMD(); // TODO@esm ASAR disabled in ESM
-
 	await makeUniversalApp({
 		x64AppPath,
 		arm64AppPath,
-		asarPath: canAsar ? asarRelativePath : undefined,
+		asarPath: asarRelativePath,
```

Comment:
This will do proper merging of the asar files and at the same time also copy the `node_modules`. No need to condition on it.